.github/workflows/provider.yaml

```diff
@@ -58,9 +58,14 @@ jobs:
 
       - name: Run lakeFS DAG
         run: |
-          docker container exec astrobfff1f_scheduler_1 airflow connections add conn_1 --conn-type=HTTP --conn-host=http://172.17.0.1:8000 --conn-extra='{"access_key_id":"${{ env.KEY }}","secret_access_key":"${{ env.SECRET }}"}'
-          docker container exec astrobfff1f_scheduler_1 airflow dags unpause lakeFS_workflow
-          docker container exec astrobfff1f_scheduler_1 airflow dags trigger lakeFS_workflow && sleep 30
+          cd astro
+          echo "add conn_1"
+          astro dev run connections add conn_1 --conn-type=HTTP --conn-host=http://172.17.0.1:8000 --conn-extra='{"access_key_id":"${{ env.KEY }}","secret_access_key":"${{ env.SECRET }}"}'
+          echo "dag unpause"
+          astro dev run dags unpause lakeFS_workflow
+          echo "dag trigger"
+          astro dev run dags trigger lakeFS_workflow
+          sleep 30
```

Comment:
why?

Reply:
- will remove the debug messages
- the multiline works the same just readable